config/settings.yml

```diff
@@ -76,6 +76,8 @@
   :concurrent_per_host: 1
   :scan_via_host: true
   :use_vim_broker: true
+:container_scanning:
+  :concurrent_per_ems: 3
```

Comment:
I'm not a fan of the existing `concurrent_per_ems` name because it doesn't say what we're limiting.  I'm ok with it because naming is hard but `concurrent_scans_per_ems` feels more useful even though it's not consistent.  I don't know.